src/partials/text-main-description.html.md

```diff
@@ -2,4 +2,4 @@
 
 ![Icon](/images/icon.png)
 
-TransforMap works towards an online platform to visualize the myriad of alternatives to the dominant economic thinking on a single mapping system. It will give everyone the opportunity to map the initiatives, communities, projects, worker-owned, self-managed, democratically organised companies and other institutions dedicated to meeting people's needs, serving the common good and/ or contributing to a sustainable way of life.
\ No newline at end of file
+TransforMap works towards an online platform to visualize the myriad of alternatives to the dominant economic thinking on a single mapping system. It will give everyone the opportunity to map the initiatives, communities, projects, worker-owned, self-managed, democratically organised companies and other institutions dedicated to meeting people's needs, serving the common good and/ or contributing to a sustainable way of life. Together with the existing maps, this will serve as a tool to connect these initiatives and strengthen them by linking both local and national efforts.
```

Comment:
instead of `national` efforts, it seems `transnational` or `global` would be more fitting here.